File: lib_common/src/main/java/com/fly/tour/common/mvp/ILoadView.java
package com.fly.tour.common.mvp;

/**
 * Description: <ILoadView><br>
 * Author:      mxdl<br>
 * Date:        2018/2/26<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
public interface ILoadView {
    //显示初始加载的View，初始进来加载数据需要显示的View
    void showInitLoadView();
    //隐藏初始加载的View
    void hideInitLoadView();
}

File: lib_common/src/main/java/com/fly/tour/common/mvp/ILoadView.kt
package com.fly.tour.common.mvp

/**
 * Description: <ILoadView><br>
 * Author:      mxdl<br>
 * Date:        2018/2/26<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
</ILoadView> */
interface ILoadView {
    //显示初始加载的View，初始进来加载数据需要显示的View
    fun showInitLoadView()

    //隐藏初始加载的View
    fun hideInitLoadView()
}
